utilities/src/main/java/io/syndesis/qe/utils/TestUtils.java

```diff
@@ -158,4 +161,37 @@ public static void sleepIgnoreInterrupt(long milis) {
             e.printStackTrace();
         }
     }
+
+    public static void updateCredentialJson(String connectionName, DBAllocation dbAllocation) {
```

Comment:
Please rename this method, it sounds confusing for me, you are not editing credentials.json file at all. Be more specific, `TestUtils.updateCredentialsJson()` is static so it is available from everywhere - it needs a good name which says what is the method used for. What about `setDatabaseCredentials`?